### DISCUSSION TITLE
マウスホイールはどう設定していますか?

### DISCUSSION THREAD

**Discussion by mtsmfm** [Yowkees/keyball#55] [Q&A] [answered]
keyball で操作を完結させたいのですが、remap で Mouse Wh を設定しても反応しません。

- 特定のキーをホイール操作扱いにする
- 特定のキーを押している間、トラックボールをホイール扱いにする

のどちらかができると嬉しいのですが、そのような方法はあるのでしょうか。

**Comment by Yowkees** [comment, marked as answer]
レイヤー3に切り替えている間、トラックボールは縦横スクロールに変わります。

**Comment by Yowkees** [comment]
Mouse Whは使ったことがなかったので、ファームの再確認が必要そうです。

私はボールのスクロール機能と、PgUp,PgDnをクリック近くのキーに割り当てて使っていて特に不便は感じていませんでした。

その他のキーでボールをスクロールに切り替えるにはファームウェアの編集とmakeが必要だと思います。

**Reply by kodaishida** [reply]
keyball61を使っているのですが、こちら現状いかがでしょうか？手持ちの最新ファームではmouse whの設定ができませんでした。
右手でペンタブも併用しており、左手側でspace+F等でホイール操作がしたく問い合わせさせていただきました。

**Comment by mtsmfm** [comment]
おお、恥ずかしながら layer 3 でホイールになることに気づいていませんでした。
ありがとうございます。
